### PR TITLE
Add continuous export configuration

### PR DESCRIPTION
**Pull request by arielshaqed**
(Doesn't do anything yet)

### CODE REVIEW

File: ddl/0012_export_continuous.down.sql [hunk 1]
```diff
@@ -0,0 +1 @@
+ALTER TABLE catalog_branches_export DROP COLUMN continuous;
```
Comment (guy-har) [inline ddl/0012_export_continuous.down.sql:1]: IF EXISTS

File: ddl/000012_export_continuous.up.sql [hunk 2]
```diff
@@ -0,0 +1,7 @@
+BEGIN;
+
+ALTER TABLE catalog_branches_export ADD COLUMN continuous BOOLEAN;
```
Comment (guy-har) [inline ddl/000012_export_continuous.up.sql:3]: IF NOT EXISTS

File: cmd/lakectl/cmd/export.go [hunk 3]
```diff
@@ -53,25 +63,33 @@ var exportConfigurationTemplate = `export configuration for branch "{{.Branch.Re
...
-
+{{.ContinuousMarker}}
 `
 
 // exportGetCmd get continuous export configuration for branch
 var exportGetCmd = &cobra.Command{
 	Use:   "get <branch uri>",
 	Short: "get continuous export configuration for branch",
 	Run: func(cmd *cobra.Command, args []string) {
+		if len(args) != 1 {
```
Comment (guy-har) [inline cmd/lakectl/cmd/export.go:74]: you could use Args for validation, same as in commitCmd

**Review by guy-har — APPROVED** [review]
looks good

**Review by arielshaqed — COMMENTED** [review]
Thanks!  Will pull once tests pass.

**Pull request merged**